src/app/[locale]/seller/reviews/[id]/page.tsx

```diff
@@ -157,15 +157,15 @@ function SellerReviews({
 
         {/* Search area */}
         <div className='flex gap-3 items-center justify-items-center py-3'>
-          <span>{t('SHARED.PIONEER_LABEL')}</span>
+          <span>{userName}</span>
           <FormControl className="flex-grow mr-2">
             <TextField
               id="search-input"
               type="text"
               variant="outlined"
               color="success"
               className="bg-none hover:bg-gray-100 w-full rounded-lg"
-              label={t('SHARED.SEARCH_REVIEWS')}
+              label={t('Search reviews by pioneer username')}
```

Comment:
Shouldn't this be `label={userName}`?

Reply:
we need to confirm from @Phillip. may be you should merged the changes first and then allow him to test and point out the changes.